lua/mason-nvim-dap/mappings/configurations.lua

```diff
@@ -0,0 +1,153 @@
+local settings = require('mason-nvim-dap.settings')
+
+local M = {}
+
+-- https://github.com/mfussenegger/nvim-dap/wiki/Debug-Adapter-installation#go-using-delve-directly
+M.go = {
+	{
+		type = 'delve',
+		name = 'Debug',
+		request = 'launch',
+		program = '${file}',
+	},
+	{
+		type = 'delve',
+		name = 'Debug test', -- configuration for debugging test files
+		request = 'launch',
+		mode = 'test',
+		program = '${file}',
+	},
+	-- works with go.mod packages and sub packages
+	{
+		type = 'delve',
+		name = 'Debug test (go.mod)',
+		request = 'launch',
+		mode = 'test',
+		program = './${relativeFileDirname}',
+	},
+}
+
+M.python = {
+	{
+		-- The first three options are required by nvim-dap
+		type = 'python', -- the type here established the link to the adapter definition: `dap.adapters.python`
+		request = 'launch',
+		name = 'Launch file',
+
+		-- Options below are for debugpy, see https://github.com/microsoft/debugpy/wiki/Debug-configuration-settings for supported options
+
+		program = '${file}', -- This configuration will launch the current file if used.
+		pythonPath = function()
+			-- debugpy supports launching an application with a different interpreter then the one used to launch debugpy itself.
+			-- The code below looks for a `venv` or `.venv` folder in the current directly and uses the python within.
+			-- You could adapt this - to for example use the `VIRTUAL_ENV` environment variable.
+			local cwd = vim.fn.getcwd()
+			if vim.fn.executable(cwd .. '/venv/bin/python') == 1 then
+				return cwd .. '/venv/bin/python'
+			elseif vim.fn.executable(cwd .. '/.venv/bin/python') == 1 then
+				return cwd .. '/.venv/bin/python'
+			else
+				return '/usr/bin/python'
+			end
+		end,
```

Comment:
I think for the most basic out of the box experience it might be a good idea to leave this one out. It's not super generalized to all virtual environments. If the user has activated a python virtual environment before opening neovim it works as expected. This makes a very strong assumption that all virtual environments are named `venv`. If the user wants to further configure `pythonPath` then they can manually to fit their needs. What do you think?

Reply:
For the most basic configuration here that works that I use is: 

```lua
      {
        type = "python",
        request = "launch",
        name = "Launch file",
        program = "${file}",
      },
```